client/src/controllers/ActionController.ts

```diff
@@ -106,4 +106,30 @@ export class ActionController extends Controller<
     if (this.element instanceof HTMLButtonElement) return;
     this.element?.select();
   }
+
+  /**
+   * Reset the field to a supplied or the field's initial value (default).
+   * Only update if the value to change to is different from the current value.
+   */
+  reset(
+    event: CustomEvent<{ value?: string }> & { params?: { value?: string } },
+  ) {
+    const target = this.element;
+    const currentValue = target.value;
+
+    const { value: newValue = '' } = {
+      value: target instanceof HTMLInputElement ? target.defaultValue : '',
+      ...event?.params,
+      ...event?.detail,
+    };
+
+    if (currentValue === newValue) return;
+
+    target.value = newValue;
+    this.dispatch('change', {
+      bubbles: true,
+      cancelable: false,
```

Comment:
```suggestion
      cancelable: false,
      prefix: '',
```

We will need to set prefix to an empty string, otherwise the event dispatched will have the name `'w-action:change'`. You can read more about this here https://stimulus.hotwired.dev/reference/controllers#cross-controller-coordination-with-events

Reply:
Done this and re-tested. Works fine.